python-avd/pyavd/_eos_designs/structured_config/overlay/ip_security.py

```diff
@@ -47,7 +47,7 @@ def ip_security(self: AvdStructuredConfigOverlay) -> dict | None:
         return strip_null_from_data(ip_security)
 
     def _append_data_plane(self: AvdStructuredConfigOverlay, ip_security: dict, data_plane_config: dict) -> None:
-        """In place update of ip_security."""
+        """In place update of ip_security for DataPlane."""
```

Comment:
Line 42 and 43 in this file already implemented the behavior advertised here as "new".

Reply:
yeah I picked up the TODO in the other file and that's probably why I thought this was new - let me change the title